Makefile

```diff
@@ -10,118 +24,172 @@ ifndef GOBIN
 	$(error GOBIN is not set)
 endif
 
-#======= Main functions =======
-macpgo:	check-go-vars
+#======= Kubernetes =======
+installrbac: ## Installs kubernetes rbac policies that are required for the operator pod to run
+	cd deploy && ./install-rbac.sh
+
+setupnamespaces: ## Creates the kubernetes namespace(s) as defined by $PGO_OPERATOR_NAMESPACE
+	cd deploy && ./setupnamespaces.sh
+
+cleannamespaces: ## Deletes the kubernetes namespace(s) as defined by $PGO_OPERATOR_NAMESPACE
+	cd deploy && ./cleannamespaces.sh
+
+bounce: ## Restarts the postgres-operator pods
+	$(PGO_CMD) --namespace=$(PGO_OPERATOR_NAMESPACE) get pod --selector=name=postgres-operator -o=jsonpath="{.items[0].metadata.name}" | xargs $(PGO_CMD) --namespace=$(PGO_OPERATOR_NAMESPACE) delete pod
+
+
+#======= Platform specific =======
+macpgo:	check-go-vars ## Builds the pgo cli binary for the mac(darwin) platform
 	cd pgo && env GOOS=darwin GOARCH=amd64 go build pgo.go && mv pgo $(GOBIN)/pgo-mac
-	env GOOS=darwin GOARCH=amd64 go build github.com/blang/expenv && mv expenv $(GOBIN)/expenv-mac
-winpgo:	check-go-vars
+	env GOOS=darwin GOARCH=amd64 go build github.com/blang/expen && mv expenv $(GOBIN)/expenv-mac
+
+winpgo:	check-go-vars ## Builds the the pgo binary for the windows platform
 	cd pgo && env GOOS=windows GOARCH=386 go build pgo.go && mv pgo.exe $(GOBIN)/pgo.exe
 	env GOOS=windows GOARCH=386 go build github.com/blang/expenv && mv expenv.exe $(GOBIN)/expenv.exe
 
-installrbac:
-	cd deploy && ./install-rbac.sh
-setup:
+
+#====== golang =======
+setup: ## Install the golang dependencies
```

Comment:
These are build dependencies in a general sense - some are golang, but mostly environment and container dependencies - documentation likely should reflect that and it be moved to the appropriate section.